miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -30,23 +30,6 @@ abstract class MiniApp internal constructor() {
     @Throws(MiniAppSdkException::class)
     abstract suspend fun listMiniApp(): List<MiniAppInfo>
 
-    /**
-     * Creates a mini app.
-     * The mini app is downloaded, saved and provides a [MiniAppDisplay] when successful.
-     * @param appId mini app id.
-     * @param miniAppMessageBridge the interface for communicating between host app & mini app
-     * @throws [MiniAppNotFoundException] when the specified mini app ID does not exist on the server
-     * @throws [MiniAppHasNoPublishedVersionException] when the specified mini app ID exists on the
-     * server but has no published versions
-     * @throws [MiniAppSdkException] when there is any other issue during fetching,
-     * downloading or creating the view.
```

Comment:
Do we need to remove those information for `@throws`?

Reply:
ah let me update those information.